.travis.yml

```diff
@@ -111,9 +121,9 @@ _helm_deploy: &helm_deploy
 
       while [ "$PODS_COUNTER" -lt "$PODS_COUNTER_MAX" ]; do
         totalpods=$(pod_status | grep -v NAME | wc -l | sed 's/ *//')
-        readypodcount=$(pod_status | grep ' True' | wc -l | sed 's/ *//')
+        readypodcount=$(pod_status | grep -e ' True' -e Succeeded | wc -l | sed 's/ *//')
```

Comment:
The reindexing is performed by a Kubernetes Job. This means that the Pod terminates with status Succeded when it finishes its job. The previous check was considering only running pods.